JsonEncodingException when call LottieAnimationView.setAnimationFromUrl

**Describe the bug**
Lottie 4.2.1
call LottieAnimationView.setAnimationFromUrl(..)，when download zip file's Content-Type is not 'application/zip'，maybe is 'application/x-zip' or "application/x-zip-compressed"，exception occurs： 
    Caused by: com.airbnb.lottie.parser.moshi.JsonEncodingException: Use JsonReader.setLenient(true) to accept malformed JSON at path $
        at com.airbnb.lottie.parser.moshi.JsonReader.syntaxError(JsonReader.java:249)
        at com.airbnb.lottie.parser.moshi.JsonUtf8Reader.checkLenient(JsonUtf8Reader.java:952)
        at com.airbnb.lottie.parser.moshi.JsonUtf8Reader.doPeek(JsonUtf8Reader.java:348)
        at com.airbnb.lottie.parser.moshi.JsonUtf8Reader.beginObject(JsonUtf8Reader.java:137)

**Screenshots**
![image](https://user-images.githubusercontent.com/19383947/143429318-2476162c-66d0-4b4f-8876-05fe0e325474.png)
Only Content-Type 'application/zip' be handled as '.ZIP' file

--- a/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
+++ b/lottie/src/main/java/com/airbnb/lottie/network/NetworkFetcher.java
@@ -107,7 +107,10 @@ public class NetworkFetcher {
       // in the result which is more useful than failing here.
       contentType = "application/json";
     }
-    if (contentType.contains("application/zip") || url.split("\\?")[0].endsWith(".lottie")) {
+    if (contentType.contains("application/zip") ||
+        contentType.contains("application/x-zip") ||
+        contentType.contains("application/x-zip-compressed") ||
+        url.split("\\?")[0].endsWith(".lottie")) {
       Logger.debug("Handling zip response.");
       extension = FileExtension.ZIP;
       result = fromZipStream(url, inputStream, cacheKey);

